checkmate/views/ui/admin.py

```diff
@@ -24,3 +41,26 @@ def get(self):
     @forbidden_view_config()
     def logged_out(self):
         return HTTPFound(location=self.request.route_url("pyramid_googleauth.login"))
+
+
+@view_defaults(
+    route_name="admin.allow_rule",
+    permission=Permissions.ADMIN,
+    renderer="checkmate:templates/admin/allow_rule.html.jinja2",
+)
+class AdminAllowRuleViews:
+    def __init__(self, request):
+        self.request = request
+
+    @view_config(request_method="GET")
+    def get(self):
+        return {}
+
+    @view_config(request_method="POST")
+    def post(self):
+        url = self.request.params["url"]
+        try:
+            allow_rule = self.request.find_service(RuleService).add_to_allow_list(url)
+        except ResourceConflict as e:
```

Comment:
Reusing the same `ResourceConflict` here from the api version for now even considering its parent being `JSONAPIException`.